src/bitcoind.js

```diff
@@ -82,7 +82,7 @@ export async function bitcoindListUnspent({ url, auth, address, addresses }) {
   } catch (e) {
     // eslint-disable-next-line no-console
     console.error("There was a problem:", e.message);
-    return e;
+    throw e;
```

Comment:
I wonder if this try catch is even necessary then.

Reply:
possibly not - this is a bugfix to actually throw the error back up the chain instead of just returning. the caller was trying to .slice on this -- which obviously failed when an Error was returned.